app/models/site_config.rb

```diff
@@ -36,18 +36,6 @@ class SiteConfig < RailsSettings::Base
   }
   field :authentication_providers, type: :array, default: %w[]
 
-  # Campaign
-  field :campaign_call_to_action, type: :string, default: "Share your project"
-  field :campaign_hero_html_variant_name, type: :string, default: ""
-  field :campaign_featured_tags, type: :array, default: %w[]
-  field :campaign_sidebar_enabled, type: :boolean, default: 0
-  field :campaign_sidebar_image, type: :string, default: nil, validates: {
-    format: { with: VALID_URL, message: URL_MESSAGE }
-  }
-  field :campaign_url, type: :string, default: nil
-  field :campaign_articles_require_approval, type: :boolean, default: 0
-  field :campaign_articles_expiry_time, type: :integer, default: 4
```

Comment:
Removing the fields will make the migration script fail

Reply:
Right. Second time in a row I do this even though I **know** it won't work. 

![image](https://user-images.githubusercontent.com/47985/114645792-4d76e680-9d04-11eb-8fd5-e4c300db2b04.png)